src/interfaces/interface.ts

```diff
@@ -0,0 +1,18 @@
+export interface ICompetency {
+	id: string,
+	faultsNo: number,
+	isSerious: boolean,
+	isDangerous: boolean
+}
+
+export interface ITestResult {
+	id?: string,
+	_candidateId: string,
```

Comment:
Don't prepend candidateId with _ here, it is inconsistent

Reply:
eh, unfortunately, it is consistent (https://github.com/dvsa/mobile-examiner-alpha-journal-service/blob/develop/test-data/journal.json - line 24) ;(. I'm not fun of it either but I think we should be consistent for us